README.md

```diff
@@ -209,6 +209,15 @@ TrailingWhitespace
 UnorderedKey
 ```
 
+`dotenv-linter` can also automatically fix warnings in the files. Currently only one kind of warnings is fixed
+(`LowercaseKey`). You should use the argument `--autofix` (or its short version `-f`) for this.
+
+```shell script
+$ dotenv-linter -f
+Unfixed: .env:2 DuplicatedKey: The BAR key is duplicated
+  Fixed: .env:3 LowercaseKey: The FOO key should be in uppercase
```

Comment:
To tell the truth, I don't like this output.
It will look a little ugly if we check many files:
```shell script
Unfixed: .env:1 KeyWithoutValue: The FOO key should be with a value or have an equal sign
Unfixed: .env:2 LeadingCharacter: Invalid leading character detected
Unfixed: .env:2 KeyWithoutValue: The  BAR key should be with a value or have an equal sign
Unfixed: .env:3 LeadingCharacter: Invalid leading character detected
Unfixed: .env:3 KeyWithoutValue: The 1 key should be with a value or have an equal sign
Unfixed: subdir/.env:1 LeadingCharacter: Invalid leading character detected
Unfixed: subdir/.env:1 KeyWithoutValue: The 2 key should be with a value or have an equal sign
  Fixed: subdir/.env:2 LowercaseKey: The foo key should be in uppercase
Unfixed: subdir/.env:2 QuoteCharacter: The value is wrapped in quotes
Unfixed: subdir/.env.common:2 ExtraBlankLine: Extra blank line detected
  Fixed: subdir/.env.common:3 LowercaseKey: The foo key should be in uppercase
Unfixed: subdir/.env.common:5 ExtraBlankLine: Extra blank line detected
Unfixed: subdir/test.env:2 UnorderedKey: The FOO key should go before the _TEST key
  Fixed: subdir/test.env:3 LowercaseKey: The var key should be in uppercase
```

I'd like to see something like this:
```
Fixed warnings:
subdir/.env:2 LowercaseKey: The foo key should be in uppercase
subdir/.env.common:3 LowercaseKey: The foo key should be in uppercase
subdir/test.env:3 LowercaseKey: The var key should be in uppercase

Unfixed warnings:
.env:1 KeyWithoutValue: The FOO key should be with a value or have an equal sign
.env:2 LeadingCharacter: Invalid leading character detected
.env:2 KeyWithoutValue: The  BAR key should be with a value or have an equal sign
.env:3 LeadingCharacter: Invalid leading character detected
.env:3 KeyWithoutValue: The 1 key should be with a value or have an equal sign
subdir/.env:1 LeadingCharacter: Invalid leading character detected
subdir/.env:1 KeyWithoutValue: The 2 key should be with a value or have an equal sign
subdir/.env:2 QuoteCharacter: The value is wrapped in quotes
subdir/.env.common:2 ExtraBlankLine: Extra blank line detected
subdir/.env.common:5 ExtraBlankLine: Extra blank line detected
subdir/test.env:2 UnorderedKey: The FOO key should go before the _TEST key
```

Reply:
Should we print corrected `warings`? :thinking: 
Linters often do not display this information, but since they work inside the version control system, I believe that it is not always relevant for us

But I too prefer the option of dividing information into blocks mentioned by  @mgrachev :rocket:

Reply:
Hmm, it's a good question, because we plan to fix all warnings.
Maybe, in the future we won't print the `Unfixed` group at all. 🤔

Reply:
I think the output in groups is not hard to implement.
But the current format is easier to handle with another command, (with pipeline).

Reply:
If it is matter.

Reply:
> I think the output in groups is not hard to implement.

Great. I'd then stay on it.

Reply:
I have changed the output format.
Also I have added some tests for this feature.